processor/transform/readme-components.ts

```diff
@@ -52,19 +52,19 @@ const coerceJsxToMd =
       };
 
       parent.children[index] = mdNode;
-    } else if (node.name === 'Image') {
-      const { position } = node;
-      const { alt = '', url, title = null } = attributes<Pick<Image, 'alt' | 'title' | 'url'>>(node);
-
-      const mdNode: Image = {
-        alt,
-        position,
-        title,
-        type: 'image',
-        url,
-      };
-
-      parent.children[index] = mdNode;
+    // } else if (node.name === 'Image') {
+    //   const { position } = node;
+    //   const { alt = '', url, title = null } = attributes<Pick<Image, 'alt' | 'title' | 'url'>>(node);
+
+    //   const mdNode: Image = {
+    //     alt,
+    //     position,
+    //     title,
+    //     type: 'image',
+    //     url,
+    //   };
+
+    //   parent.children[index] = mdNode;
```

Comment:
@kellyjosephprice i commented out some of your image stuff because it was gettin' rull weird with my changes and i figured we could work it out later

Reply:
I think this means 'complexImages' won't render as the image widget in the editor, right?

Reply:
Same for embeds, I think we need to convert them from JSX to mdast/slate-y type nodes for the editor to pick them up.

Reply:
it works in the editor playground, but i haven't checked in a project yet (doing so now)

Reply:
If you change the playground source to:
```
<Image align="center" className="border" url="https://drastik.ch/wp-content/uploads/2023/06/blackcat.gif" />
```

does it load as a widget or as JSX?

Reply:
in the playground, yeah
<img width="919" alt="image" src="https://github.com/readmeio/markdown/assets/20567510/2fb6ed52-7a09-4c4a-9ea6-30aeb2f828f6">

<img width="991" alt="image" src="https://github.com/readmeio/markdown/assets/20567510/7706d7d5-9294-400a-a0b1-ab7395bde6e7">

but in the dash editor not so much, goddammit

Reply:
OH I think it's falling through in to here: https://github.com/readmeio/markdown/blob/ef1c17b7b4fee02d72dc0d24fce40ef79039346b/processor/transform/readme-components.ts#L97-L111

Reply:
I _think_ we should add embeds to this map: https://github.com/readmeio/markdown/blob/ef1c17b7b4fee02d72dc0d24fce40ef79039346b/processor/transform/readme-components.ts#L9-L19

And then they'll get picked up in the editor?